src/main/kotlin/com/pubnub/api/PNConfiguration.kt

```diff
@@ -26,43 +26,33 @@ import javax.net.ssl.X509ExtendedTrustManager
  * A storage for user-provided information which describe further PubNub client behaviour.
  * Configuration instance contains additional set of properties which
  * allow performing precise PubNub client configuration.
+ *
  */
 open class PNConfiguration(
-    userId: UserId,
-    enableSubscribeBeta: Boolean = false
+    var userId: UserId
 ) {
+    var enableEventEngine: Boolean = false
```

Comment:
Maybe it could be placed down below e.g. just above
var subscribeKey: String =
Maybe small comment could be added informing about purpose of this property?

Reply:
Done